utils/func_utils.py

```diff
@@ -1,4 +1,39 @@
+from typing import Any
+from dotenv import load_dotenv
+import os
 import bcrypt
+import jwt
+import datetime
 
-def get_password_hash(password:str) -> str: 
-    return bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
\ No newline at end of file
+
+def get_password_hash(password: str) -> str:
+    salt = bcrypt.gensalt()
+    hashed = bcrypt.hashpw(password.encode('utf-8'), salt)
+    return hashed.decode('utf-8')
+
+
+def check_password(password: str, hashed: str) -> bool:
+    return bcrypt.checkpw(password.encode('utf-8'), hashed.encode('utf-8'))
+
+
+load_dotenv()
+SECRET_KEY = os.getenv('SECRET_KEY', 'DEFAULT_KEY')
```

Comment:
The token is not in the repository. I added a default value, 'DEFAULT_KEY,' to ensure the tests pass when submitting my PR. Since the secret key is stored in the .env file, which is not accessible from the repository, we can share it through a secure channel (e.g., email).

Reply:
Nice. The answer I was looking for